solutions/banking/outputs.tf

```diff
@@ -0,0 +1,44 @@
+output "project_id" {
+  description = "ID of the created project."
```

Comment:
I would expend on the descriptions here - eg: indicating this is a watsonx project